src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,60 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type MovieProp = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<MovieProp> = ({ movie }) => {
+  const {
+    description,
+    imdbUrl,
+    imgUrl,
+    title,
+  } = movie;
+
+  return (
+    <div className="card" data-cy="Movie">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img
+            data-cy="MovieImage"
+            src={imgUrl}
+            alt="Film logo"
```

Comment:
```suggestion
            alt="Movie logo"
```
lets change 'Film' to 'Movie' for consistency :)